### PR TITLE
bagmetv es6-basics files

### PR DESCRIPTION
**Pull request by bahmetdev**

### CODE REVIEW

File: homeworks/es6-basics/bagmetv/main.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+
+
+ function Item (id, name, quote) {
+    this.id = id;
+    this.name = name;
+    this.quote = quote;
+} 
+
+UserCart.prototype = new Cart ();
```
Comment (denis-zavgorodny) [inline homeworks/es6-basics/bagmetv/main.js]: Зачем создавать отдельно объект, можно просто связать прототипы `UserCart.prototype` и `Cart.prototype`

File: homeworks/es6-basics/bagmetv/main.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    // getting total quote/amount
+    this.amount = function () {
+        var total_quote = 0;
+        for (var i = 0; i < this.goods.length; i++) {
+            total_quote += this.goods[i].quote;
+        }
+        return total_quote;
+    };
+
+    // updateQnt
```
Comment (denis-zavgorodny) [inline homeworks/es6-basics/bagmetv/main.js]: Метод не реализован

**Review by denis-zavgorodny — CHANGES_REQUESTED** [review]

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Please, fix your solution!

File: homeworks/es6-basics/bagmetv/main.js [hunk 3]
```diff
@@ -13,50 +13,63 @@ Cart.prototype.add = function (item) {
...
-} 
+    this.amount = amount;
+}
 
-UserCart.prototype = new Cart ();
+function UserCart () {}
 
-function UserCart () {
+    // linking prototypes
+UserCart.prototype = new Cart ();
```
Comment (denis-zavgorodny) [inline homeworks/es6-basics/bagmetv/main.js]: `UserCart.prototype = Cart.prototype` можно просто связать прототипы, не обязательно создавать новый объект

File: homeworks/es6-basics/bagmetv/.jshintrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (denysdovhan) [inline homeworks/es6-basics/bagmetv/.jshintrc:1]: drop this file

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Why did you delete `README`? `.DS_Store` should be ignored

**Comment by bahmetdev** [comment]
Hi! Sorry, my fault. Please specify how to resolve those conflicting files.

**Comment by maxkharena** [comment]
You should resolve git conflict 
"how to" links:
https://githowto.com/resolving_conflicts
https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/
http://gitready.com/advanced/2009/02/25/keep-either-file-in-merge-conflicts.html

**Comment by denysdovhan** [comment]
We close this PR since it's not completed, conflict still not resolved and a deadline was expired.

**Pull request closed without merging**